gateway/1.4/modules/ROOT/pages/flex-conn-manage-jenkins.adoc

```diff
@@ -380,7 +316,7 @@ NOTE: You can only apply Connected Mode policies.
 == Deploying an API Instance
```

Comment:
This heading is a gerund, while others like it are imperative. These should be consistent.